### PR TITLE
Jitosol

### PR DESCRIPTION
**Pull request by bergarces**

### CODE REVIEW

File: packages/adapters-library/src/adapters/beefy/sdk/config.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { fetchIchiBalanceBreakdown } from './unwrap/ichi'
 import { fetchPendleEquilibriaBalanceBreakdown } from './unwrap/pendle_equilibria'
 import { fetchSolidlyBalanceBreakdown } from './unwrap/solidly'
 
-export const chainIdMap: Record<Chain, string> = {
+export const chainIdMap: Partial<Record<Chain, string>> = {
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/beefy/sdk/config.ts:21]: If set to `EvmChain` it'll require changing `Chain` to `EvmChain` in every Beefy adapter using it. This is way easier.

File: packages/adapters-library/src/adapters/integration.test.ts [hunk 2]
```diff
@@ -247,6 +248,10 @@ const allTestCases: Record<Protocol, Record<string, TestCase[]>> = {
     ['pool']: iZiSwapPoolTestCases,
   },
 
+  [Protocol.Jito]: {
+    ['jitosol']: jitoJitosolTestCases,
+  },
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/integration.test.ts:253]: This was added automatically but snapshots are currently not supported.

File: packages/adapters-library/src/adapters/aave-v3/products/rewards/aaveV3RewardsAdapter.ts [hunk 3]
```diff
@@ -52,7 +52,7 @@ export class AaveV3RewardsAdapter implements IProtocolAdapter {
     includeInUnwrap: false,
   }
 
-  private INCENTIVES_CONTRACT_ADDRESSES = {
+  private INCENTIVES_CONTRACT_ADDRESSES: Partial<Record<Chain, string>> = {
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/aave-v3/products/rewards/aaveV3RewardsAdapter.ts:55]: Preferable than also manually excluding Solana in the if statement below.

File: packages/adapters-library/src/core/constants/chains.ts
Comment (bergarces) [inline packages/adapters-library/src/core/constants/chains.ts:1]: Open to suggestiongs, but this is the least disruptive way I found of adding Solana without having to do `Exclude<Chain, typeof Chain.Solana>` all over the place.

File: packages/adapters-library/src/core/constants/AVERAGE_BLOCKS_PER_DAY.ts [hunk 4]
```diff
@@ -11,4 +11,5 @@ export const AVERAGE_BLOCKS_PER_DAY: Record<Chain, number> = {
   [Chain.Arbitrum]: 320278,
   [Chain.Avalanche]: 41413,
   [Chain.Linea]: 42698,
+  [Chain.Solana]: 1, // TODO Add real blocks per day for Solana
```
Comment (bergarces) [inline packages/adapters-library/src/core/constants/AVERAGE_BLOCKS_PER_DAY.ts:14]: Not needed for Solana yet as we cannot query based on block number.

**Review by jpsains — APPROVED** [review]

**Pull request merged**